### DISCUSSION TITLE
Support for windows x86?

### DISCUSSION THREAD

**Discussion by moth1995** [AntiMicroX/antimicrox#909] [Q&A]
hi guys, just a question related if its possible to give support for previous windows releases on x86, i know is basically legacy but many people still uses those version, even considering the older games works better on those old versions due to older version of directx (yes i know there are some workarounds for some games but not all of them works)

**Comment by pktiuk** [comment]
> if its possible to give support for previous windows releases on x86

Everything is possible, but who will do it?

**Reply by pktiuk** [reply]
I guess you could try to compile it by yourself on your machine, later you could just add an legacy windows build to release workflow, but this part may be a bit problematic.

**Reply by moth1995** [reply]
Thanks! my question was more related to if there were some issue due to newer libraries or stuff related i will give it a go during this week

**Reply by pktiuk** [reply]
It mainly depends on SDL2 library, but they seem to have also older builds
